ui/app/AppLayouts/Communities/popups/ExportControlNodePopup.qml

```diff
@@ -4,147 +4,288 @@ import QtQuick.Layouts 1.15
-            width: scroll.availableWidth
-            spacing: 20
-            Paragraph {
-                text: qsTr("For a Status Community to function, it needs to have a single control node running. This installation of Status Desktop is currently the %1 community control node. To move the %1 control node to another device: ").arg(root.communityName)
+        Paragraph {
+            text: d.hasSyncedDesktopDevices ? qsTr("Any of your synced <b>desktop</b> devices can be the control node for this Community:")
+                                            : qsTr("You don’t currently have any <b>synced desktop devices</b>. You will need to sync another desktop device before you can move the %1 control node to it. Does the device you want to use as the control node currently have Status installed?").arg(root.community.name)
+        }
+
+        Loader {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.leftMargin: Style.current.bigPadding
+            Layout.rightMargin: Style.current.bigPadding
+            sourceComponent: d.hasSyncedDesktopDevices ? devicesInstructions : noDevicesInstructions
+        }
+    }
+
+    footer: StatusDialogFooter {
+        rightButtons: ObjectModel {
+            StatusButton {
+                text: qsTr("Close")
+                onClicked: root.close()
             }
-            ColumnLayout {
+        }
+    }
+
+    Component {
+        id: devicesInstructions
+
+        ColumnLayout {
+            spacing: Style.current.padding
+
+            Rectangle {
                 Layout.fillWidth: true
-                spacing: 4
-                Paragraph {
-                    text: qsTr("1. Copy your Community’s private key:")
-                }
-                StatusBaseInput {
-                    id: privateKeyTextArea
-                    Layout.fillWidth: true
-                    multiline: true
-                    edit.readOnly: true
-                    text: root.privateKey
-                    rightComponent: CopyButton {
-                        onClicked: {
-                            privateKeyTextArea.edit.selectAll()
-                            privateKeyTextArea.edit.copy()
-                            privateKeyTextArea.edit.deselect()
+                Layout.leftMargin: -40
+                Layout.rightMargin: -40
+                Layout.preferredHeight: devicesView.implicitHeight
+                Layout.fillHeight: true
+                color: Theme.palette.baseColor2
+
+                Rectangle {
+                    anchors.fill: parent
+                    anchors.leftMargin: -parent.Layout.leftMargin
+                    anchors.rightMargin: -parent.Layout.rightMargin
+                    anchors.topMargin: 28
+                    anchors.bottomMargin: 28
+                    color: Theme.palette.indirectColor4
+                    radius: Style.current.radius
+                    clip: true
+
+                    StatusListView {
```

Comment:
Not clear enough from the design but it seems the devices could be clasified in 2 different lists, eligible or not eligible. At least, there is a separator line in between them that is missing here!

See the design:
![Screenshot 2023-09-15 at 16 19 33](https://github.com/status-im/status-desktop/assets/97019400/4cd55254-e670-4c9f-87a2-f7f35639de38)

Reply:
Yeah but it's kinda incomplete; there might be also desktop devices that are not eligible so we decided to leave the separator out